frontend/src/views/Crops/CropsModal.tsx

```diff
@@ -752,8 +762,15 @@ function CropsModal({
                     <Select
                       isRequired
                       label="# of plants per acre"
-                      items={plantsPerAcre}
-                      selectedKey={formData.numberOfPlantsPerAcre}
+                      items={berryQuantities.map((ele) => ({
```

Comment:
Please define this list higher up, as creating a new list or object triggers React rerenders.